lucien-ios/LoginScreen/RootViewController.swift

```diff
@@ -64,20 +65,19 @@ final class RootViewController: UIViewController, GIDSignInDelegate, GIDSignInUI
     }
 
     func showDashboard() {
-        let dashboardViewController = DashboardViewController()
-        let dashboardNavigationController = UINavigationController(rootViewController: dashboardViewController)
-        present(dashboardNavigationController, animated: true, completion: nil)
+        dashboardViewModel.getDashboard {
+            guard let dashboardData = DashboardViewModel.dashboardData else { return }
```

Comment:
This shouldn't be happening in the view controller - exposing the `dashboardData` property defeats the purpose of MVVM